src/Store.js

```diff
@@ -3,7 +3,6 @@ import EventEmitter from 'eventemitter3';
 
 const CHANGE = 'change';
 const ERROR = 'error';
-const STATE = Symbol();
```

Comment:
I am not sure what's the practical need for an import here. As it's just an attribute, it will end up being used as `this._state_` anyway in this case.